tests/tests/test_timetable.py

```diff
@@ -14,14 +14,27 @@ class _TrainScheduleDetails:
     id: int
     train_name: str
     departure_time: int
-    train_path: int
+    arrival_time: float
+    path_id: int
+    timetable_id: int
+    rolling_stock_id: int
+    initial_speed: int
+    labels: Iterable[str]
+    allowances: Iterable[dict]
+    speed_limit_tags: str
+    comfort: str
+    options: dict
+    power_restriction_ranges: Iterable[dict]
+    mechanical_energy_consumed: Iterable[dict]
```

Comment:
The `Iterables` don't generally support `len(x)`, which we use later on. It should probably be changed to `List`